test/responses_unit_test.go

```diff
@@ -109,18 +107,40 @@ func weMakeAnyCallTo(client /* algod/indexer */, endpoint string) (err error) {
 				GenesisID:        sParams.GenesisID,
 				Genesishash:      sParams.GenesisHash,
 				LastRound:        uint64(sParams.FirstRoundValid),
-				MinFee:           81560,
+				MinFee:           sParams.MinFee,
 			}
 		case "GetApplicationByID":
 			response, err = algodC.GetApplicationByID(10).Do(context.Background())
 		case "GetAssetByID":
 			response, err = algodC.GetAssetByID(10).Do(context.Background())
+		case "PendingTransactionInformation":
+			response, _, err = algodC.PendingTransactionInformation("transaction").Do(context.Background())
+		case "GetPendingTransactions":
+			var total uint64
+			var top []types.SignedTxn
+			total, top, err = algodC.PendingTransactions().Do(context.Background())
+			response = models.PendingTransactionsResponse{
+				TopTransactions:   top,
+				TotalTransactions: total,
+			}
+		case "GetPendingTransactionsByAddress":
+			var total uint64
+			var top []types.SignedTxn
+			total, top, err = algodC.PendingTransactionsByAddress("address").Do(context.Background())
+			response = models.PendingTransactionsResponse{
+				TopTransactions:   top,
+				TotalTransactions: total,
+			}
```

Comment:
:point_up: Old tests supported by the new verify function.

:point_down: new endpoints